modules/installation-and-upgrade/pages/container-deployment/suma/server-migration-suma.adoc

```diff
@@ -8,7 +8,8 @@ endif::[]
 
 // We need to figure out which SUMA versions prior to the container release can or should be migrated. Something like any version prior to yyyy.mm and later than.
 
-To migrate a {productname} 4.3 Server to a container, a new machine is required.
+To migrate a {productname} 4.3 Server to a container, you require a new machine with {sle-micro} 5.5 and [litaral]``mgradm`` installed. 
```

Comment:
```suggestion
To migrate a {productname} 4.3 Server to a container, you require a new machine with {sle-micro} 5.5 and [literal]``mgradm`` installed. 
```
Reading the bug and Michele's reply I've the impression that you'd better recommend to use  a new machine with no previous OS installed. And in the near future (5.0.2?), `mgradm` will be installed automatically if you use the official installation source.
I think you can also change the IMPORTANT note accordingly.